src/AdaProx.jl

```diff
@@ -272,6 +272,43 @@ function stepsize(rule::OurRule, (gamma1, gamma0), x1, grad_x1, x0, grad_x0)
     return (gamma, sigma), (gamma, gamma1)
 end
 
+
+
+struct OurRulePlus{R}
```

Comment:
Man, we really need to improve the naming convention 😅

Should we just use “author initials + year + rule” all over the module?

Reply:
Maybe in a separate PR though, I can do that

Reply:
I agree it is a mess. The name OurRulePlus is terrible indeed. Your suggestion sounds like a great idea.